scripts/generate_etl_figure.py

```
@@ -0,0 +1,229 @@
+        ha='center',
+        va='bottom',
+        fontsize=9,
+        color=color,
+        fontweight=weight,
+    )
+
+# ——— Active-segment label —————————————————————————————————————————————————————
+ax.text(
+    (ql_a + qu_a) / 2,
+    1.5,
+    'active segment\n' r'$b_{n=1} = 1$',
```

Ruff (0.15.21)

[warning] Implicitly concatenated string literals on one line

(ISC001)